$OpenBSD: patch-src_libpiano_crypt_h,v 1.3 2014/05/20 14:51:46 dcoppa Exp $

piano: Fix include guard of crypt.h
(upstream git commit b534a3bf3133a2fc7f0024dbe8df77410dba6111)

--- src/libpiano/crypt.h.orig	Tue May 20 16:22:13 2014
+++ src/libpiano/crypt.h	Tue May 20 16:23:11 2014
@@ -21,8 +21,8 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE O
 THE SOFTWARE.
 */
 
-#ifndef _CRYPH_H
-#define _CRYPT_H
+#ifndef SRC_LIBPIANO_CRYPT_H_O832IVGK
+#define SRC_LIBPIANO_CRYPT_H_O832IVGK
 
 #ifdef __FreeBSD__
 #define _GCRYPT_IN_LIBGCRYPT
@@ -33,4 +33,4 @@ char *PianoDecryptString (gcry_cipher_hd_t, const char
 		size_t * const);
 char *PianoEncryptString (gcry_cipher_hd_t, const char *);
 
-#endif /* _CRYPT_H */
+#endif /* SRC_LIBPIANO_CRYPT_H_O832IVGK */
